core/flyout_vertical.ts

```diff
@@ -233,29 +233,32 @@ export class VerticalFlyout extends Flyout {
     for (let i = 0, item; (item = contents[i]); i++) {
       if (item.type === 'block') {
         const block = item.block;
-        const allBlocks = block!.getDescendants(false);
+        if (block == undefined || block == null) {
```

Comment:
This check is redundant, block == null checks both null and undefined. We would both checks with strict equality i.e. block === undefined || block === null

Reply:
In this case `if (block)` will do fine, because we aren't trying to distinguish between the two cases.

Reply:
If I just add if (block) than it shows "'block' is possibly 'undefined'." error. So, I added condition if (block == null) as suggested by rachna.